### DISCUSSION TITLE
typescript correctness of type 'group'

### DISCUSSION THREAD

**Discussion by Duckinm** [payloadcms/payload#336] [General]
I have created some group fields that provide no 'name' but it has no hint to let me correct, as I read the doc it is a required field. Should it be fixed?

Attaching:
![Screen Shot 2564-10-06 at 6 20 22 PM](https://user-images.githubusercontent.com/29366271/136193475-edbd2302-7a2e-4f88-bab6-432e1372759b.png)

**Comment by DanRibbens** [comment]
Good catch, the type should include `name: string'` for that field as well as some others I'm noticing don't have it. 
The name is required by the schema validation step when Payload starts, but a warning before then is nicer. Expect to see this fixed in the next release. 

Thanks @Duckinm!
